services/outputhost/cgcache.go

```diff
@@ -264,7 +264,7 @@ func newConsumerGroupCache(destPath string, cgDesc shared.ConsumerGroupDescripti
 		cfgMgr:                   h.cfgMgr,
 	}
 
-	cgCache.checkSingleCGVisibleCache = cache.NewLRUWithInitialCapacity(1, 1 << 6)
+	cgCache.checkSingleCGVisibleCache = cache.NewLRUWithInitialCapacity(1, 1<<6)
```

Comment:
It is big but since it is just the maxSize i think it is fine, since the initial size is still 1.. One more reason for making this 64 is i think @GuillaumeBailey wants to guarantee that this extent will definitely be in the cache to prevent the metadata call..